---
string value  # XML value representation of the parameter
